api/app_sph_lms/models.py

```diff
@@ -1,379 +1,448 @@
+        )
+    is_active = models.BooleanField(default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ('trainee', 'course')
+        verbose_name = "CourseTrainee"
+        verbose_name_plural = "CourseTrainees"
+        db_table = "app_sph_lms_course_trainees"
+
+    def __str__(self):
+        return (
+            "Trainee: "
+            + str(self.trainee)
+            + " | "
+            + "Course: "
+            + str(self.course)
+        )
+
+
+class LearningPath(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[MinLengthValidator(3)]
+    )
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    image = models.CharField(max_length=255, null=True)
+    is_active = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        verbose_name = "LearningPath"
+        verbose_name_plural = "LearningPaths"
+        db_table = "app_sph_lms_learning_paths"
+
+    def __str__(self):
+        return self.name
+
+
+class LearningPathCourse(models.Model):
+    learning_path = models.ForeignKey(LearningPath, on_delete=models.CASCADE)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    course_order = models.IntegerField()
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        unique_together = ('learning_path', 'course')
+        verbose_name = "LearningPathCourse"
+        verbose_name_plural = "LearningPathCourse"
+        db_table = "app_sph_lms_learning_path_courses"
+
+    def __str__(self):
+        return (
+            "Learning Path: " +
+            str(self.learning_path) +
+            " | " +
+            "Course: " +
+            str(self.course)
+        )
+
+
+class LearningPathCategory(models.Model):
+    learning_path = models.ForeignKey(LearningPath, on_delete=models.CASCADE)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        unique_together = ('learning_path', 'category')
+        verbose_name = "LearningPathCategory"
+        verbose_name_plural = "LearningPathCategory"
+        db_table = "app_sph_lms_learning_path_categories"
+
+    def __str__(self):
+        return (
+            "Learning Path: " +
+            str(self.learning_path) +
+            " | " +
+            "Category: " +
+            str(self.category)
+        )

```

Comment:
```suggestion

```
There is no need for this one, after you declaer the relations in the model, django will automatically generate this table in db

Please refer to docs to see the usage of creating data with relation, so you can apply  it in your serializer as well
https://docs.djangoproject.com/en/4.2/topics/db/examples/many_to_many/